docs/community/exhibition-checklist.md

```diff
@@ -0,0 +1,116 @@
+* **When**: Typically 3-4 months _before_ event (check event website for deadlines)
+* **Where**: Event/conference website
+* **Why**: In order to get a booth to exhibit, we need to apply and pay any fees, if required
+* **How**:
+    * Events we frequently attend have copy+paste text available in FOSS@MAGIC Google Drive
+    * However, most info we submit is also found on our [public website](https://fossrit.github.io) (see [_About_](https://fossrit.github.io/about/) and [_Projects_](https://fossrit.github.io/projects/))
+
+### Add exhibited projects to FOSS@MAGIC website
+
+Any project we exhibit at an event needs a project profile on the FOSS@MAGIC website.
+See [Website - How to add projects](/infra/website "Scroll down to 'How to add projects' section") for how to do this.
+
+### Add event to FOSS@MAGIC website
+
+Any event, conference, or show we attend needs an event profile on the FOSS@MAGIC website.
+The website does not support recurring events, so each individual event should get its own profile.
+See [Website - How to create events](/infra/website "Scroll down to 'How to create events' section") for how to do this.
+
+### Pack inventory for exhibition booth
+
+* **What**: Pack supplies, hardware, and other inventory for an exhibition booth
+* **Who**: Booth volunteers
+* **When**: 1-2 days before event travel
+* **Where**: In RIT MAGIC Suite / Prof. Stephen Jacob's office
+* **Why**: To prepare for the exhibit and **test all hardware BEFORE event to avoid on-site problems on event day**
+* **How**: Assemble the following items:
+    * Tablecloth (x1)
+    * Printed brochures/handouts (as available, [ask FOSS@MAGIC contacts](/about/contact) for guidance)
+    * Laptops (x2)
+    * Monitors (x2)
+    * USB keyboard (x1)
+    * USB mouse (x1)
+    * Any physical projects to be exhibited
+
+#### Inventory tips
+
+* Exhibit laptops typically need access by whoever is at the booth.
+  Suggest turning off automatic screen-lock or creating a temporary shared password/account.
+* If monitors not available, use laptops for demoing.
+  Note this means many hands on a device, so make sure device owner is okay with this first.
+
+### Set up exhibition booth
+
+* **What**: Set up inventory and booth for event
+* **Who**: Booth volunteers
+* **When**: Day of event
+* **Where**: Event venue
+* **Why**: To make an awesome display and get people to ask questions about what we do
+* **How**:
+    1. Spread tablecloth out, if event does not provide one
+    1. Set up each laptop with a monitor on opposite ends of table
+    1. Display [FOSS@MAGIC website](https://fossrit.github.io) on one monitor and other project work on other monitor
+    1. Set up hardware projects in center of table
+    1. If missing equipment or need supplies from outside the venue, [contact FOSS@MAGIC staff](/about/contact) immediately
+    1. Make loose plans with all volunteers for who will be present at table when, when lunch break is, etc.
```

Comment:
A script outline on FOSS@MAGIC would be super useful.

Reply:
@Nolski Like an elevator pitch? Do you think this is sufficient?

https://fossrit.github.io/about/